### PR TITLE
feat(i18n): add Spanish language support

### PR DESCRIPTION
**Pull request by upupnoah**
Added Spanish translations to the application, including localization for app content, common phrases, external resources, language settings, and shortcuts.

### CODE REVIEW

File: locales/lang/es.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "langs.en": "English",
+  "langs.ja": "日本語",
+  "langs.zh-CN": "简体中文",
```
Comment (Innei) [inline locales/lang/es.json:4]: You should add `langs.es` in every `lang/*.json`

**Review by Innei — CHANGES_REQUESTED** [review]

File: locales/lang/es.json (hunk 1 above)
Comment (upupnoah) [reply locales/lang/es.json:4]: okey

Comment (upupnoah) [reply locales/lang/es.json:5]: > You should add `langs.es` in every `lang/*.json`

Already added.

**Comment by Innei** [comment]
Please resolves conflicts

**Comment by upupnoah** [comment]
> Please resolves conflicts
Conflict has been resolved.

**Comment by upupnoah** [comment]
> Please resolves conflicts

Conflict has been resolved.

**Review by Innei — APPROVED** [review]

**Comment by Innei** [comment]
Thank you!

**Pull request merged**